packages/application-generic/src/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -105,6 +105,31 @@ Handlebars.registerHelper(
   }
 );
 
+// based on: https://gist.github.com/DennyLoko/61882bc72176ca74a0f2
+Handlebars.registerHelper(
+  HandlebarHelpersEnum.NUMBERFORMAT,
+  function (number, options) {
+    if (isNaN(number)) {
+      return number;
+    }
+
+    const dl = options.hash.decimalLength || 2;
+    const ts = options.hash.thousandsSep || ',';
+    const ds = options.hash.decimalSep || '.';
+
+    const value = parseFloat(number);
+
+    const re = '\\d(?=(\\d{3})+' + (dl > 0 ? '\\D' : '$') + ')';
+
+    const num = value.toFixed(Math.max(0, ~~dl));
```

Comment:
```suggestion

    const decimalLength = options.hash.decimalLength || 2;
    const thousandsSep = options.hash.thousandsSep || ',';
    const decimalSep = options.hash.decimalSep || '.';

    const value = parseFloat(number);

    const numberFormattingRegex = '\\d(?=(\\d{3})+' + (dl > 0 ? '\\D' : '$') + ')';

    const formattedNumber = value.toFixed(Math.max(0, ~~dl));
```

A small suggestion to improve readability here :)